README.md
# tsadm webapp
TSAdm django webapp


static/ChangeLog.txt
2015-02-12 14:04  jrms

	* [r12628] sbin/site-repo-create: TSAdm: script update

2015-02-11 19:36  jrms

	* [r12623] tsadm_tests/regression/tests/sites.py: TSAdm: regression
	  tests updates

2015-02-11 18:18  jrms

	* [r12620] templates/site/admin.html: TSAdm: buttons l&f

2015-02-11 17:48  jrms

	* [r12617] static/css/tsadm.css: TSAdm: CSS update

2015-02-11 14:00  jrms

	* [r12606] static/css/tsadm.css: TSAdm: buttons CSS update

2015-02-11 06:31  jrms

	* [r12603] static/css/tsadm.css, templates/git/home.html,
	  templates/inc/navbar.html, templates/mysql/home.html,
	  templates/rsync/home.html, templates/site/dashboard.html,
	  templates/site/navbar.info.html: TSAdm: HTML l&f changes

2015-02-11 05:20  jrms

	* [r12602] BUGS.txt: TSAdm: BUGS update

2015-02-11 05:09  jrms

	* [r12598] libexec/jobq/git/checkout-branch: TSAdm: fix bug in
	  git.checkout-branch

2015-02-11 05:02  jrms

	* [r12597] tsadm/admin/views.py: TSAdm: fix db maint

2015-02-11 04:59  jrms

	* [r12596] tsadm/admin/urls.py, tsadm/admin/views.py,
	  tsadm/db/__init__.py, tsadm/db/sql.py: TSAdm: db maint

2015-02-11 04:03  jrms

	* [r12595] templates/inc/admin_navbar.html,
	  templates/user/admin.html, tsadm/db/__init__.py, tsadm/db/sql.py,
	  tsadm/user/urls.py, tsadm/user/views.py: TSAdm: user admin - list
	  all

2015-02-11 01:11  jrms

	* [r12594] tsadm/admin/urls.py, tsadm/admin/views.py: TSAdm:
	  activity log limit in URL

2015-02-11 00:59  jrms

	* [r12593] templates/user/home.html, tsadm/db/__init__.py,
	  tsadm/db/schema/tsadm.tbl.sql, tsadm/db/schema/user.tbl.sql,
	  tsadm/db/sql.py, tsadm/db/upgrade/10000003.sql,
	  tsadm/user/__init__.py, tsadm/wapp.py: TSAdm: user last seen

2015-02-10 23:55  jrms

	* [r12592] static/css/tsadm.css, templates/admin,
	  templates/admin.html, templates/admin/activity.html,
	  templates/admin/home.html, templates/inc/admin_navbar.html,
	  tsadm/admin, tsadm/admin/__init__.py, tsadm/admin/urls.py,
	  tsadm/admin/views.py, tsadm/db/__init__.py,
	  tsadm/db/schema/activity_log.tbl.sql,
	  tsadm/db/schema/tsadm.tbl.sql, tsadm/db/sql.py,
	  tsadm/db/upgrade/10000002.bash, tsadm/db/upgrade/10000002.sql,
	  tsadm/settings.py, tsadm/urls.py, tsadm/user/__init__.py,
	  tsadm/views.py, tsadm/wapp.py: TSAdm: admin module and activity
	  log

2015-02-10 16:51  jrms

	* [r12591] TODO.txt: TSAdm: TODO update

2015-02-06 17:25  jrms

	* [r12586] ., tsadm/settings.py: TSAdm: fix offline file path

2015-02-06 17:14  jrms

	* [r12585] templates/inc/head.html, tsadm/wapp.py: TSAdm: html base
	  tag

2015-02-06 17:13  jrms

	* [r12584] templates/git/home.html,
	  templates/site/navbar.info.html: TSAdm: fix links target
	  attribute

2015-02-06 16:53  jrms

	* [r12583] static/css/tsadm.css, templates/inc/validator.html:
	  TSAdm: small/obsesive minor HTML change in footer

2015-02-06 14:20  jrms

	* [r12582] BUGS.txt, TODO.txt: TSAdm: BUGS and TODO list updates

2015-02-05 22:26  jrms

	* [r12581] static/css/tsadm.css, templates/inc/head.html,
	  templates/inc/validator.html, tsadm/settings.py, tsadm/wapp.py:
	  TSAdm: CSS3 validator

2015-02-05 20:04  jrms

	* [r12580] Makefile, mk/django, mk/django/Makefile: TSAdm: split
	  django Makefile

2015-02-05 19:19  jrms

	* [r12577] tsadm/settings.py, tsadm/wapp.py: TSAdm: offline path
	  from settings.py

2015-02-05 19:16  jrms

	* [r12576] templates/error/offline.html, tsadm/wapp.py: TSAdm:
	  offline mode

2015-02-05 17:46  jrms

	* [r12570] static/css/tsadm.css: TSAdm: HTML monospace font-family

2015-02-05 17:42  jrms

	* [r12569] tsadm/site/__init__.py: TSAdm: fix bug when site.env is
	  not found

2015-02-05 17:42  jrms

	* [r12568] tsadm_tests/regression/__init__.py: TSAdm: use new fixed
	  address

2015-02-05 17:34  jrms

	* [r12567] tsadm/site/__init__.py, tsadm/wapp.py: TSAdm: fix bug
	  when site.env is not found

2015-02-05 17:32  jrms

	* [r12566] Makefile: TSAdm: django listen fixed address

2015-01-23 16:23  jrms

	* [r12469] templates/inc/head.html: TSAdm: HTML5 charset definition

2015-01-16 18:04  jrms

	* [r12417] sbin/site-repo-create: TSAdm: improve sbin script

2015-01-16 18:02  jrms

	* [r12416] sbin/site-repo-create: TSAdm: improve sbin script

2015-01-16 17:58  jrms

	* [r12415] sbin/slave-setup: TSAdm: fix script

2015-01-16 17:26  jrms

	* [r12414] sbin/site-repo-create: TSAdm: update sbin script

2015-01-16 17:23  jrms

	* [r12413] sbin/site-repo-create: TSAdm: update sbin script

2015-01-15 18:47  jrms

	* [r12409] sbin/slave-setup: TSAdm: improve scripts

2015-01-15 17:59  jrms

	* [r12402] sbin/slave-setup: TSAdm: bug fixes

2015-01-15 17:55  jrms

	* [r12401] sbin/slave-setup: TSAdm: bug fix

2015-01-15 17:24  jrms

	* [r12399] mk/_utils/merge.py: TSAdm: improve mk script

2015-01-15 05:18  jrms

	* [r12397] tsadm/git/views.py: TSAdm: improve code

2015-01-15 04:57  jrms

	* [r12393] tsadm/wapp.py: TSAdm: set start timestamp when start is
	  called

2015-01-15 04:54  jrms

	* [r12392] templates/git/home.html: TSAdm: improve commands objects
	  build and their UI

2015-01-15 04:52  jrms

	* [r12391] tsadm/git/jobq.py, tsadm/git/views.py,
	  tsadm/jobq/cmd.py, tsadm/wapp.py: TSAdm: set start timestamp at
	  __init__ time

2015-01-15 03:38  jrms

	* [r12390] templates/git/home.html: TSAdm: typo fix

2015-01-15 03:37  jrms

	* [r12389] templates/git/home.html, tsadm/git/jobq.py: TSAdm:
	  git.fetch frontend command

2015-01-15 03:14  jrms

	* [r12387] libexec/jobq/git/fetch, tsadm/git/views.py: TSAdm: git
	  post-receive fetch metadata

2015-01-15 02:32  jrms

	* [r12385] libexec/jobq/git/fetch-tags, tsadm/git/views.py: TSAdm:
	  git post-receive fetch tags

2015-01-15 02:32  jrms

	* [r12384] libexec/git/post-receive: TSAdm: do not use SSL if in
	  devmode

2015-01-15 02:30  jrms

	* [r12383] tsadm/wapp.py: TSAdm: set devmode if running via
	  manager.py

2015-01-15 02:30  jrms

	* [r12382] tsadm/log.py: TSAdm: send log to stderr if in devmode

2015-01-15 00:54  jrms

	* [r12381] Makefile: TSAdm: make django-runserver

tsadm/urls.py
# $Id: urls.py 12762 2015-04-07 22:25:30Z jrms $

from django.conf.urls import patterns, include, url

urlpatterns = patterns('',
    url(r'^$', 'tsadm.site.views.index', name='index'),
    url(r'^soft-info/$', 'tsadm.views.soft_info', name='soft-info'),

    url(r'^cmd/confirm/$', 'tsadm.views.cmd_confirm', name='cmd_confirm'),
    url(r'^cmd/exec/$', 'tsadm.views.cmd_exec', name='cmd_exec'),

    url(r'^admin/', include('tsadm.admin.urls', namespace='admin')),
    url(r'^site/', include('tsadm.site.urls', namespace='site')),
    url(r'^git/', include('tsadm.git.urls', namespace='git')),
    url(r'^files/', include('tsadm.rsync.urls', namespace='rsync')),
    url(r'^db/', include('tsadm._mysql.urls', namespace='mysql')),
    url(r'^jobq/', include('tsadm.jobq.urls', namespace='jobq')),
    url(r'^help/', include('tsadm.help.urls', namespace='help')),
    url(r'^user/', include('tsadm.user.urls', namespace='user')),
    url(r'^slave/', include('tsadm.slave.urls', namespace='slave')),
    url(r'^asb/', include('tsadm.ansible.urls', namespace='ansible')),
)
